### DISCUSSION TITLE
How to test white label apps locally?

### DISCUSSION THREAD

**Discussion by ashanitelasoft** [codemagic-ci-cd/codemagic-docs#2196] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

i have setup white lable app on codemagic which works fine. but i need to do the same concept on locally just like codemagic does during build time. like changing bundle id , change assets file for both android and IOS. and i wanted to do the same on locally. is there a codemagic cli or tool build white label apps locally?

### Expected results

-

### Actual results

-

**Comment by icarusdust** [comment]
Hi @ashanitelasoft setting bundle IDs, changing icons etc. are all done through `sed`, `cp` etc. commands and these can be executed on your local machine as expected. When it comes to building and publishing your app, you still can install Codemagic CLI tools and execute the same commands from your machine. [Here](https://docs.codemagic.io/knowledge-codemagic/codemagic-cli-tools/) is the documentation

**Comment by icarusdust** [comment]
I am closing the thread due to inactivity. You can re-open it by replying to it, or start a new thread here or in our community [Discord channel](https://discord.com/invite/pefznye93R).
